src/Spectre.Database/App.config

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This config file is repeated. We could consider extracting it's functionality into a service class (under an interface) that will provide connections for desired DBs.

Reply:
It's fine by me as long as it is not repeated :)

Reply:
config file is now deleted in database.tests project